app/src/main/java/cm/aptoide/pt/repository/request/GetStoreWidgetsRequestFactory.java

```diff
@@ -61,4 +62,12 @@ public GetStoreWidgetsRequest newStoreWidgets(String url, boolean googlePlayServ
         resources, windowManager, clientUniqueId, googlePlayServicesAvailable, partnerId,
         accountMature, filters, systemService, versionCodeProvider);
   }
+
+  public GetStoreWidgetsRequest newStoreWidgets(String url, boolean googlePlayServicesAvailable,
```

Comment:
Same as AdsRepository, would create a class in your flavour where i would add your new methods.